src/shared.ts

```diff
@@ -23,6 +23,7 @@ export type ConnectElementHTMLName =
   | "stripe-connect-issuing-cards-list"
   | "stripe-connect-financial-account"
   | "stripe-connect-financial-account-transactions"
+  | "stripe-connect-capital-overview"
```

Comment:
Can you link to the API review in the PR description? Otherwise looks good 👍

Reply:
https://docs.google.com/document/d/1jdjW8OLe0F8eU_cFhrKXTrjpt0ZCSIQPkl3OLtkflPE/edit#heading=h.nbrl5773k5jx